### DISCUSSION TITLE
[help] how to test  cors rule ?

### DISCUSSION THREAD

**Discussion by huyinghuan** [gavv/httpexpect#123] [Q&A] [answered]
i test online example:

```
	e := httpexpect.New(t, "https://api.mydomain.com")
	request := e.Get("/api/info")
	response := request.Expect()
	response.Status(200)
	log.Println(response.Body().Raw())
```

now i want to test my cors rule,  just like invoke api "https://api.mydomain.com/api/info"  in "https://www.mydomain.com" domain

how can i write the test unit? Thanks for your help.

**Comment by gavv** [comment, marked as answer]
Hi! If you're asking how to change domain for specific request, take a look at this example: https://github.com/gavv/httpexpect#subdomains-and-per-request-url
